sql/wsrep_sst.cc

```diff
@@ -946,6 +946,8 @@ const std::string get_allowed_pwd_chars() { return g_allowed_pwd_chars; }
 static void generate_password(std::string *password, int size) {
   std::stringstream ss;
   bool srnd;
+  ss << "yx9!A-";
```

Comment:
Maybe it is worth considering something like
`constexpr const char *prefix = "yx9!A-";`
`ss << prefix;`
`ss << strlen(prefix)`

That will get rid of hardcodded magic values and give  compiler the chance to compile time optimize strlen().

Reply:
Done.